README.md

```diff
@@ -1,17 +1,78 @@
-# How to use Open Virtual Networking with Kubernetes
+# Open Virtual Networking (OVN)
 
-This document describes how to use Open Virtual Networking with Kubernetes
+OVN provides network virtualization to containers.  In the "overlay" mode,
+OVN can create a logical network amongst containers running on multiple hosts.
+In this mode, OVN programs the Open vSwitch instances running inside your
+hosts.  These hosts can be bare-metal machines or vanilla VMs.
+
+## OVN via kubernetes daemonsets
+
+The easiest way to use OVN as the networking plugin (via CNI) for kubernetes
+is via kubernetes daemonsets. This currently only works for Linux nodes.
```

Comment:
@shettyg The ovn dev preview that I have been working on supports both windows and linux nodes. I am not involved with the windows parts. Thought you might like to know.

Reply:
Okay. We will update the doc once it is well supported.